### ISSUE TITLE
Found a liitle bug or typo in the documentation.

### ISSUE THREAD

**Issue by Mrutyunjaya1994** [mdn/content#33189]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

### What specific section or headline is this issue about?

However, ({ a, b } = { a: 1, b: 2 }) is valid, as is const { a, b } = { a: 1, b: 2 }.

### What information was incorrect, unhelpful, or incomplete?

However, ({ a, b } = { a: 1, b: 2 }) is valid, as is const { a, b } = { a: 1, b: 2 }.
Here it says , (In my point of view),  ({ a, b } = { a: 1, b: 2 })  is equal to const { a, b } = { a: 1, b: 2 }.
but when i'm trying to modify this variable a & b, then it is fine. Then how it is possible?? how can i modify a const variable.

for example, 
({ a, b } = { a: 1, b: 2 });

console.log(a, b)  =>> 1 , 2

a = 100; b = 200;

console.log(a, b) =>> 100, 200

so i think this should be, ==>>   ({ a, b } = { a: 1, b: 2 }) -->>  is equal to --->>  let { a, b } = { a: 1, b: 2 }.

//...............

Because -- const {a, b} = {a: 10, b: 100}; 
console.log(a, b) =>> 10, 100
a = 100; b = 200;   =>> Not possible because these are const

but in this case ({ a, b } = { a: 1, b: 2 }) ;
how it is possible to modify a and b;
a = 100; b = 200 ==> NO ERROR 

so i think , ({ a, b } = { a: 1, b: 2 }) -->>  is equal to --->>  const { a, b } = { a: 1, b: 2 }. is a wrong statement.

### What did you expect to see?

i want to see, 

({ a, b } = { a: 1, b: 2 }) -->>  is equal to --->>  let { a, b } = { a: 1, b: 2 }.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/operators/destructuring_assignment`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/operators/destructuring_assignment/index.md
* Last commit: https://github.com/mdn/content/commit/e3faa375b0179de77a5eff00074e3d168a0a904c
* Document last modified: 2023-10-04T23:23:49.000Z

</details>

**Comment by Mrutyunjaya1994** [comment]
Thank You josh for your clarification, i got confused , i thought, documentation indicating that both are same , i mean 
({ a, b } = { a: 1, b: 2 })   ===    const { a, b } = { a: 1, b: 2 }.
But from your clarification i found both are not equal. But it looks like (for a beginner), both are same.
Thank You.

**Comment by Josh-Cena** [comment]
You are right that they are not the same. You just misunderstood what this sentence is meant to say
